packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -322,7 +324,6 @@ function getRuntime() {
 		);
 		console.warn(`[${PACKAGE_NAME}] Your project will use Node.js 18 as the runtime instead.`);
 		console.warn(`[${PACKAGE_NAME}] Consider switching your local version to 18.`);
-		return 'nodejs18.x';
```

Comment:
I think we need to keep an empty `return` here to pass the test

Reply:
But I passed the same in the getRuntime function, aight though

Reply:
I mean we need to return an empty `return;`, otherwise it hits the next if condition below. Another option is to change the if condition below as an else-if

Reply:
done!